src/table/utils/__tests__/handle-accessiblity.spec.ts

```diff
@@ -309,41 +320,41 @@ describe('handle-accessibility', () => {
     it('should not call blur when currentTarget contains relatedTarget', () => {
       containsRelatedTarget = true;
 
-      handleAccessibility.handleFocusoutEvent(evt, shouldRefocus, keyboard);
+      handleAccessibility.handleFocusoutEvent(focusoutEvent, shouldRefocus, keyboard);
       expect(keyboard.blur).not.toHaveBeenCalled();
     });
 
     it('should not call blur when shouldRefocus is true', () => {
       shouldRefocus.current = true;
 
-      handleAccessibility.handleFocusoutEvent(evt, shouldRefocus, keyboard);
+      handleAccessibility.handleFocusoutEvent(focusoutEvent, shouldRefocus, keyboard);
       expect(keyboard.blur).not.toHaveBeenCalled();
     });
 
-    it('should not call blur when keyboard.enabled is falsey', () => {
+    it('should not call blur when keyboard.enabled is falsy', () => {
```

Comment:
```suggestion
    it('should not call blur when keyboard.enabled is false', () => {
```

Reply:
It might be undefined when it is not true, so falsy should cover both false and undefined

Reply:
I do not follow. I see that `keyboard.enabled = false;` below. So 'should not call blur when keyboard.enabled is false' makes sense to me. And I do not see `keyboard.enabled = undefined`.

Reply:
To be honest what Li said is true. However falsy covers both false and undefined but in our cases undefined hasn't been considered as I checked. It was a written testcase, I just corrected the dictation and was not aware of whether it could be false/undefined. I think we better keep 'false' instead of falsy.

Reply:
yeah it should probably be undefined, since the real enabled is undefined and not false AFAIK. This is not very important though so for this PR it is not super relevant